### PR TITLE
Issue #13345: Enabled MutableExceptionCheckExamplesTest

### PR DESCRIPTION
**Pull request by Prathamesh-007**
Begin #13345:
Enabled MutableExceptionCheckExamplesTest.java

### CODE REVIEW

**Comment by Prathamesh-007** [comment]
Initial PR was giving too many conflicts. In this PR I have followed the guidelines given in [this](https://checkstyle.org/beginning_development.html).

Will make sure this doesn't happen again

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/mutableexception/Example1.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+// xdoc section -- start
+class Example1 extends Exception {
+  private int code; // OK, class name doesn't match with default pattern
+
+  public Example1() {
+    code = 1;
+  }
+}
+
+class FirstMyException extends Exception {
```
Comment (MANISH-K-07) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/mutableexception/Example1.java:22]: 'My' here is redundant and looks weird. Please change `FirstMyException` --> `FirstException`
Same for all

**Review by MANISH-K-07 — COMMENTED** [review]
@Prathamesh-007 , please squash and rebase to single commit

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/mutableexception/Example1.java (hunk 1 above)
Comment (Prathamesh-007) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/mutableexception/Example1.java]: Done

**Comment by Prathamesh-007** [comment]
> @Prathamesh-007 , please squash and rebase to single commit

Done

**Comment by romani** [comment]
GitHub, generate website

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**